offchain/authority-claimer/src/claimer.rs

```diff
@@ -84,22 +89,22 @@ where
     type Error = ClaimerError<B, D, T>;
 
     async fn start(mut self) -> Result<(), Self::Error> {
-        trace!("Starting the authority claimer loop");
+        debug!("Starting the authority claimer loop");
         loop {
             let rollups_claim = self
                 .broker_listener
                 .listen()
                 .await
                 .context(BrokerListenerSnafu)?;
-            trace!("Got a claim from the broker: {:?}", rollups_claim);
+            debug!("Got a claim from the broker: {:?}", rollups_claim);
 
             let is_duplicated_rollups_claim = self
                 .duplicate_checker
                 .is_duplicated_rollups_claim(&rollups_claim)
                 .await
                 .context(DuplicatedClaimSnafu)?;
             if is_duplicated_rollups_claim {
-                trace!("It was a duplicated claim");
+                info!("It was a duplicated claim");
```

Comment:
Please add the claim data to the log

```suggestion
                info!("Duplicate claim detected: {:?}", rollups_claim);
```
Same as https://github.com/cartesi/rollups-node/pull/551#discussion_r1709636097

Reply:
Done